draft-ietf-capport-api.md

```diff
@@ -196,6 +201,23 @@ Author:
 Change controller:
 : IETF
 
+## Captive Portal API Keys Registry
+
+IANA is asked to create and maintain a new registry called "Captive Portal API Keys", which will reserve JSON keys for use in Captive Portal API data structures. The initial contents of this registry are provided in {{json-keys}}.
+
+Each entry in the registry contains the following fields:
+
+Key:
+: The JSON key being registered, in string format.
+
+Type: 
+: The type of the JSON value to be stored, as one of the value types defined in {{RFC8259}}.
+
+Description:
+: A brief description explaining the meaning of the value, how it might be used, and/or how it should be interpreted by clients.
+
+New assignments for Captive Portal API Keys Registry will be administered by IANA through Expert Review {{RFC8126}}.
```

Comment:
i think it might be worthwhile to have that distinction. In fact, based on some discussions i've had with WFA, they may want to add some WFA specific key/value which could be considered "more authoritative" than values requested by some individual organization.

Reply:
Added some text giving expert guidance